apps/laboratory/tsconfig.json

```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "jsx": "preserve",
-    "noEmit": true
+    "noEmit": true,
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
since we are at it other aliases might be cool, like `@components`

Reply:
I don't really remember the exact reasons but the `@components` approach might be bit problematic at some situations. It can easy conflicted with package names with organizations like `@web3modal/..`. When pointing to internal components the `@/` generally distinguished better.